megazords/full/src/lib.rs

```diff
@@ -19,6 +19,7 @@ pub use push;
 // TODO: Drop this dependency once android-components switches to using `rust_log_forwarder` for
 // log forwarding.
 pub use rc_log_ffi;
+pub use remote_settings;
```

Comment:
Do we want to expose this to firefox-ios as well? I think that is best for a follow up PR, but the steps are in https://mozilla.github.io/application-services/book/howtos/adding-a-new-component.html#the-swift-bindings

Reply:
Ideally, yes. I am happy to do it here or as a follow-up. Would it be worth seeking input on the API from the iOS team as well?